modules/vba_documents/app/swagger/vba_documents/v0/controller_swagger.rb

```diff
@@ -92,7 +93,10 @@ class ControllerSwagger
             key :description, 'ID as returned by a previous create upload request'
             key :required, true
             key :example, '6d8433c1-cd55-4c24-affd-f592287a7572'
-            key :type, :string
+            schema do
+              key :type, :string
+              key :format, :uuid
```

Comment:
Does this not render correctly in swagger UI?

Reply:
type has to be nested under schema.
my changes work locally (against dev portal running locally)